chrome/content/zotero/tabs.js

```diff
@@ -633,14 +615,12 @@ var Zotero_Tabs = new function () {
 		if (tabIndexToFocus !== null) {
 			const nextTab = this._tabs[tabIndexToFocus];
 			// There may be duplicate tabs - in normal tab array and in pinned tabs
-			// So to get the right one, fetch all tabs with a given id and filter out one
-			// that's visible
+			// Go through all candidates and try to focus the visible one
 			let candidates = document.querySelectorAll(`[data-id="${nextTab.id}"]`);
 			for (let node of candidates) {
-				if (node.offsetParent) {
-					node.focus();
-					return;
-				}
+				node.focus();
+				// Visible tab was found and focused
+				if (document.activeElement == node) return;
```

Comment:
Can you explain this? The idea here is that if a tab is off-screen, it doesn't become the active element?